### ISSUE TITLE
No feedback when submitting inquiry form

### ISSUE THREAD

**Issue by sofer** [foundersandcoders/oursite#209]
We really do need some sort of feedback while we are awaiting a response from our feedback form. 

Also, I am not entirely sure about the alert box.

**Comment by rub1e** [comment]
Agreed - for a while the multiple submissions were doing my head in. But then I got perspective and made my peace with it.

Of course, what we're *actually* going to do is redesign the entire site root and branch

**Comment by des-des** [comment]
heres the form
https://github.com/foundersandcoders/oursite/blob/master/_harp/_layout.ejs#L25
into
https://github.com/foundersandcoders/oursite/blob/master/_harp/_partials/contact.ejs#L24
main.js comes in here
https://github.com/foundersandcoders/oursite/blob/master/_harp/_layout.ejs#L27
is here
https://github.com/foundersandcoders/oursite/blob/master/_harp/js/main.js.

When I click, nothing happens ... until ...
![image](https://cloud.githubusercontent.com/assets/12845233/18106034/bbdfc382-6ef9-11e6-9ab3-b187c9446561.png)

Here we define global sendMail object
https://github.com/foundersandcoders/oursite/blob/master/_harp/js/main.js#L288

and here we attach it to the form onsubmit
https://github.com/foundersandcoders/oursite/blob/master/_harp/_partials/contact.ejs#L30
I think the return false prevents a reload ...

**Comment by des-des** [comment]
Im guessing that the email get picked up here
https://github.com/foundersandcoders/mail-fac/blob/master/server.js#L14

**Comment by sofer** [comment]
Correct. I have just added you to hello@ recipients, so if you want to test it, you can do so.

**Comment by iteles** [comment]
Moving to priority 4 as this is a UX piece but the content is higher priority for the content refactor

**Comment by des-des** [comment]
removing from milestone but think this has high priority in backlog

**Comment by rebeccaradding** [comment]
@sofer @des-des as we are no longer using the EOI, is this still relevant?

**Comment by des-des** [comment]
@rebeccaradding I think this is the enquiry form, not the EOI?
